0000-account-name-system.md

```diff
@@ -0,0 +1,114 @@
+- Proposal Name: `account_name_system`
+- Start Date: 2019-07-21
+- NEP PR: [nearprotocol/neps#0000](https://github.com/nearprotocol/neps/pull/0000)
+
+# Summary
+[summary]: #summary
+
+The Account Name System is a hierarchical and decentralized naming system for accounts, contracts and applications running on the NEAR blockchain.
```

Comment:
"accounts, contracts and applications" -> "accounts, contracts, and applications"